src/System.Net.Http/src/ILLinkTrim.xml

```diff
@@ -1,5 +1,8 @@
 <linker>
   <assembly fullname="System.Net.Http">
+    <type fullname="System.Net.Http.HttpClient">
```

Comment:
What does this code do?  Just wondering...

Reply:
There's a linker tool that runs as a post-compile step and tree shakes, removing unused internals and privates. The tests rely via reflection on this property's setter not being removed, so this entry in the config file read by the tool essentially keeps the property rooted and safe from removal.